sdk/storage/src/adls_gen2/filesystem/requests/list_filesystems_builder.rs

```diff
@@ -194,25 +194,27 @@ where
     C: Client,
 {
     pub async fn finalize(self) -> Result<ListFilesystemsResponse, AzureError> {
-        let mut uri = format!("{}/?resource=account", self.client().filesystem_uri(),);
+        let uri = format!("{}/?resource=account", self.client().filesystem_uri(),);
 
-        if let Some(nm) = TimeoutOption::to_uri_parameter(&self) {
-            uri = format!("{}&{}", uri, nm);
-        }
+        //TODO: Reenable uri parameters
```

Comment:
Why is this commented out?

Reply:
I haven't touched the adls_gen2 code (aside for obvious changes to make it compile). We will address them in a separate PR (this one is already big as it is).